gprofiler/profilers/php.py

```diff
@@ -2,7 +2,9 @@
 # Copyright (c) Granulate. All rights reserved.
 # Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
 #
-import fcntl
+# Disabled code block - Not supported on Windows - Alternative required
```

Comment:
This profiler doesn't support Windows now - so instead of modifying it, you can edit `gprofiler/profilers/__init__.py` to skip importing it at all. If it's not imported - it won't be enabled (as `register_profiler` won't be called)

Reply:
Done